aspnetcore/includes/blazor-prerendering.md

```diff
@@ -46,7 +49,7 @@ Where `JSRuntime.InvokeAsync` is called, `ElementRef` is only used in `OnAfterRe
     private string infoFromJs;
     private ElementReference myElem;
 
-    protected override async Task OnAfterRenderAsync()
+    protected override async Task OnAfterRenderAsync(bool firstRender)
     {
         // TEMPORARY: Currently we need this guard to avoid making the interop
```

Comment:
This temporary code should be removed. `OnAfterRenderAsync` is no longer called during prerendering and the `IComponentContext` service has been removed.